docs/section11/Rule_11-11.md

```diff
@@ -0,0 +1,45 @@
+# Service_Water_Heating - Rule 11-11  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-11  
+
+**Rule Description:** For buildings that will have no service water-heating loads, no service water-heating shall be modeled in baseline building model
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, c
+
+**Evaluation Context:** B-RMD each SWH BAT
+**Data Lookup:**   
+
+**Function Call:** 
+- **get_component_by_id**
+- **get_SHW_equipment_associated_with_each_SWH_bat**  
+- **get_SHW_types_and_SHW_use**
+
+**Applicability Checks:**
+- Each SHW use type is applicable if there are NO SHW loads
```

Comment:
do we care to be more specific about checking if `ServiceWaterHeatingUse.use == 0`?